install_playground.sh

```diff
@@ -3,7 +3,47 @@ cd interface
 yarn install
 
 # install and run backend
-cd ../backend && python3 -m venv venv
-source ./venv/bin/activate
-pip install wheel
-pip install -r requirements.txt
\ No newline at end of file
+cd ../backend
```

Comment:
This assumes Conda is installed, which is not always the case and adds another step. Can we use virtualenv instead?

Reply:
That's what Diart requires:

```
conda install portaudio pysoundfile ffmpeg -c conda-forge
```

Reply:
I'll try asking the creator of the library if that would be possible, but Miniconda is also available for those who don't have much disk memory to spare.